File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+    //Open the Markdown file.
+    document.Open("Input.md");
+    //Save as a Word document.
+    document.Save("Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+'Create a Word document instance.
+Using document As WordDocument = New WordDocument()
+    'Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited
+    'Open the Markdown file.
+    document.Open("Input.md")
+    'Save as a Word document.
+    document.Save("Sample.docx")
+End Using
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+//Create a Word document instance.
+using (WordDocument document = new WordDocument())
+{
+    //Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+    //Open the Markdown file.
+    document.Open("Input.md");
+
+    //Save as a Word document to the MemoryStream.
+    MemoryStream stream = new MemoryStream();
+    await document.SaveAsync(stream, FormatType.Docx);
+    //Save the stream as a Word document in local machine.
+    Save(stream, "Sample.docx");
+    //Please refer to the following link to save the Word document in UWP platform
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Create a Word document instance.
+using (WordDocument document = new WordDocument())
+{
+    //Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+    //Open the Markdown file.
+    document.Open("Input.md");
+
+    //Save as a Word document to the MemoryStream.
+    MemoryStream outputStream = new MemoryStream();
+    document.Save(outputStream, FormatType.Docx);
+    outputStream.Position = 0;
+    //Download as a Word document in the browser.
+    return File(outputStream, "application/msword", "Sample.docx");
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Create a Word document instance.
+using (WordDocument document = new WordDocument())
+{
+    //Hook the event to customize the image while importing Markdown.
+    document.MdImportSettings.ImageNodeVisited += MdImportSettings_ImageNodeVisited;
+    //Open the Markdown file.
+    document.Open("Input.md");
+
+    //Save as a Word document to the MemoryStream.
+    MemoryStream outputStream = new MemoryStream();
+    document.Save(outputStream, FormatType.Docx);
+    //Save the stream as a file in the device and invoke it for viewing. 
+    Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    //Please download the helper files from the following link to save the stream as a file and open the file for viewing in Xamarin platform
```

Comment:
//Please download the helper files from the following link to save the stream as a file and open the file for viewing in the Xamarin platform.